astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
Maybe use `min` and `max` instead of `lower_limit` and `upper_limit` for all these min / max range constraints?
It's shorter and just as clear, no?